src/providers/VoiceFocusProvider/docs/useVoiceFocus.stories.mdx

```diff
@@ -0,0 +1,106 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import { useVoiceFocus } from '../';
+
+<Meta title="SDK Hooks/useVoiceFocus" />
+
+# useVoiceFocus
+
+The `useVoiceFocus` hook returns a function transforming a normal device to an Amazon voice focus device, and the support state of voice focus status.
```

Comment:
Request you to check the comment for `VoiceFocusProvider` here and below on `addVoiceFocus` documentation line.

Reply:
updated